jobs/validate.yaml

```diff
@@ -725,3 +725,46 @@
       - run-venv:
           COMMAND: |
             bash jobs/validate/sriov-spec $snap_version $series $channel
+
+- job:
+    name: 'validate-ck-cilium'
+    node: runner-validate
+    description: |
+      Validates CK edge with Cilium for n-1 releases.
+    project-type: matrix
+    scm:
+      - k8s-jenkins-jenkaas
+    wrappers:
+      - default-job-wrapper
+      - ci-creds
+    properties:
+      - block-on-infra
+      - build-discarder:
+          num-to-keep: 3
+    triggers:
+        - timed: "@weekly"
+    parameters:
+      - charms-edge
+    axes:
+      - axis:
+          type: slave
+          name: node
+          values:
+            - runner-validate
+      - axis:
+          type: user-defined
+          name: snap_version
+          values:
+            - 1.27/edge
+            - 1.26/edge
```

Comment:
Remove 1.26/edge b/c there's no cilium charm in this channel.  We may need to remember somehow that once there's a 1.28/edge channel to just keep 1.27/edge here as well
```suggestion
```